CHANGELOG.md

```diff
@@ -1117,6 +1128,58 @@ Other minor changes
   _≐′_ : Pred A ℓ₁ → Pred A ℓ₂ → Set _
   ```
 
+* Added new proofs in `Relation.Unary.Properties`:
+  ```
+  ⊆-reflexive : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐_ _⊆_
+  ⊆-antisym : Antisymmetric {A = Pred A ℓ} _≐_ _⊆_
+  ⊆-min : Min {B = Pred A ℓ} _⊆_ ∅
+  ⊆-max : Max {A = Pred A ℓ} _⊆_ U
+  P⊂Q⇒P⊆Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊂_ _⊆_
+  ⊂-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊂_ _⊂_ _⊂_
+  ⊂-⊆-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊂_ _⊆_ _⊂_
+  ⊆-⊂-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊆_ _⊂_ _⊂_
+  ⊂-respʳ-≐ : (_⊂_ {A = A} {ℓ₁} {ℓ₂}) Respectsʳ _≐_
+  ⊂-respˡ-≐ : (_⊂_ {A = A} {ℓ₁} {ℓ₂}) Respectsˡ _≐_
+  ⊂-resp-≐ : (_⊂_ {A = A} {ℓ}) Respects₂ _≐_
+  ⊂-irrefl : Irreflexive {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐_ _⊂_
+  ⊂-antisym : Antisymmetric {A = Pred A ℓ} _≐_ _⊂_
+  ⊂-asym : Asymmetric {A = Pred A ℓ} _⊂_
+  ∅-⊆′ : (P : Pred A ℓ) → ∅ ⊆′ P
+  ⊆′-U : (P : Pred A ℓ) → P ⊆′ U
+  ⊆′-refl : Reflexive (_⊆′_ {A = A} {ℓ})
+  ⊆′-reflexive : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐′_  _⊆′_
+  ⊆′-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊆′_ _⊆′_ _⊆′_
+  ⊆′-antisym : Antisymmetric {A = Pred A ℓ} _≐′_ _⊆′_
+  ⊆′-min : Min {B = Pred A ℓ} _⊆′_ ∅
+  ⊆′-max : Max {A = Pred A ℓ} _⊆′_ U
+  P⊂′Q⇒P⊆′Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊂′_ _⊆′_
+  ⊂′-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊂′_ _⊂′_ _⊂′_
+  ⊂′-⊆′-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊂′_ _⊆′_ _⊂′_
+  ⊆′-⊂′-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _⊆′_ _⊂′_ _⊂′_
+  ⊂′-respʳ-≐′ : (_⊂′_ {A = A} {ℓ₁} {ℓ₂}) Respectsʳ _≐′_
+  ⊂′-respˡ-≐′ : (_⊂′_ {A = A} {ℓ₁} {ℓ₂}) Respectsˡ _≐′_
+  ⊂′-resp-≐′ : (_⊂′_ {A = A} {ℓ}) Respects₂ _≐′_
+  ⊂′-irrefl : Irreflexive {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐′_ _⊂′_
+  ⊂′-antisym : Antisymmetric {A = Pred A ℓ} _≐′_ _⊂′_
+  P⊆Q⇒P⊆′Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊆_ _⊆′_
+  P⊆′Q⇒P⊆Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊆′_ _⊆_
+  P⊂Q⇒P⊂′Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊂_ _⊂′_
+  P⊂′Q⇒P⊂Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊂′_ _⊂_
+  ≐-refl : Reflexive {A = Pred A ℓ} _≐_
+  ≐-sym : Sym {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐_ _≐_
+  ≐-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _≐_ _≐_ _≐_
+  ≐′-refl : Reflexive {A = Pred A ℓ} _≐′_
+  ≐′-sym : Sym {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐′_ _≐′_
+  ≐′-trans : Trans {A = Pred A ℓ₁} {B = Pred A ℓ₂} {C = Pred A ℓ₃} _≐′_ _≐′_ _≐′_
+  P≐Q⇒P≐′Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐_ _≐′_
+  P≐′Q⇒P≐Q : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐′_ _≐_
```

Comment:
Missing additions in `Relation.Unary.Relation.Binary.Equality`?

Reply:
The module `Relation.Unary.Relation.Binary.Equality` is new in this release. Looking at the changelog, I thought the contents of new modules weren’t included in the ‘Other minor changes’ section. The definitions `≐-isEquivalence` and `≐′-isEquivalence` are also not included in the changelog. Shall I add `≐-setoid` and `≐′-setoid` together with those to the changelog?

Reply:
Ah sorry, I thought they were added last time. In which, yes leave them out :+1: